server/lib/get_user_database.js

```diff
@@ -0,0 +1,23 @@
+const pgClient = require('./pg_client');
+
+
+const getUserDatabase = (connection, userId, callback) => {
+  const client = pgClient(connection);
```

Comment:
GENERAL COMMENT (so can be applied to all functions requiring a database connection)

What would happen if, instead of passing the connection to the function, we passed a client to it. Then we could choose what client to pass, which would already be connected to the databse. This would remove repetitive code from all these functions.